composer.json

```diff
@@ -19,12 +18,12 @@
         "issues" : "https://github.com/Bigpoint/monolog-creator/issues"
     },
     "require": {
-        "php": ">=5.3.3",
-        "monolog/monolog": "1.*",
-        "predis/predis": "1.*"
+        "php": ">=8.1",
+        "monolog/monolog": "3.*",
+        "predis/predis": "2.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "6.*"
+        "phpunit/phpunit": "9.*"
```

Comment:
According to https://phpunit.de/supported-versions.html it should be phpunit 10.

Reply:
PHPunit 10 is still in development and not yet released ;)

https://packagist.org/packages/phpunit/phpunit

Reply:
Ah right. I didn't pay attention to the release date.